File: zmusic-bridge/src/main/java/me/zhenxin/zmusic/entity/MusicInfo.java
package me.zhenxin.zmusic.entity;

/**
 * 消息信息
 *
 * @author 真心
 * @email qgzhenxin@qq.com
 * @since 2022/7/24 12:01
 */
public class MusicInfo {
    /**
     * 歌名
     */
    private String name;
    /**
     * 歌手
     */
    private String singer;
    /**
     * 歌词
     */
    private String lyric;
    /**
     * 当前时间
     */
    private Integer currentTime;
    /**
     * 最大时间
     */
    private Integer maxTime;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSinger() {
        return singer;
    }

    public void setSinger(String singer) {
        this.singer = singer;
    }

    public String getLyric() {
        return lyric;
    }

    public void setLyric(String lyric) {
        this.lyric = lyric;
    }

    public Integer getCurrentTime() {
        return currentTime;
    }

    public void setCurrentTime(Integer currentTime) {
        this.currentTime = currentTime;
    }

    public Integer getMaxTime() {
        return maxTime;
    }

    public void setMaxTime(Integer maxTime) {
        this.maxTime = maxTime;
    }
}

File: common/src/main/kotlin/me/zhenxin/zmusic/common/modules/api/data/MusicInfo.kt
package me.zhenxin.zmusic.common.modules.api.data

/**
 * 音乐信息
 *
 * @author 真心
 * @since 2021/8/5 12:54
 * @email qgzhenxin@qq.com
 */
data class MusicInfo(
    val id: String,
    val name: String,
    val singer: String,
    val albumName: String,
    val albumImage: String,
    val duration: Int
)
